sphinx/environment/collectors/asset.py

```diff
@@ -140,7 +140,6 @@ def process_doc(self, app, doctree):
 def setup(app):
     # type: (Sphinx) -> Dict
     app.add_env_collector(ImageCollector)
-    app.add_env_collector(DownloadFileCollector)
```

Comment:
Note: I have to deprecate `DownloadFileCollector`.

Reply:
I don't know which is better way to implement this by Domain or EnvironmentCollector.
@shimizukawa what do you think?